README.md

```diff
@@ -56,6 +56,7 @@ The current version of the plugin supports next commands:
  - **`:CMakeSelectBuildType`** changes the cmake build type with argument passed and call **`:CMake`**.
  - **`:CMakeInfo`** creates a window with CMake information.
  - **`:CMakeClean`** cleans the project (it is equal of the execution `make clean`).
+ - **`:CMakeKitsReload`** reloads the CMake Kits
```

Comment:
Why do we need to have this additional command? From my point of view the plugin API already overcomplicated ( I have a plan to deprecate at least some options and don't want to introduce new commands if it is not needed).

From my point of view we need to reload cmake kits automatically before `:CMake` command if it is needed.

This new command covers only one specific case when user works with cmake kits. Moreover user can forget to call this command.

Reply:
Because then I'd need to implement caching, this was the easier choice.

I don't want to read and parse the JSON file every time the user runs `CMake` if it never changes so I just added the command to _explicitly_ reload the configs.

I will implement the cache, then, not a problem.